### PR TITLE
fix(perception): fix perception docs

### PR DESCRIPTION
**Pull request by vividf**
## Description
There are several pages in perception docs that are broken on the website. 
This PR fixes the issue including the wrong path and schema error.

## Related links

**Parent Issue:**

- Link

## Notes for reviewers

None.

## Interface changes

None.

## Effects on system behavior

None.

### CODE REVIEW

File: perception/autoware_object_merger/schema/data_association_matrix.schema.json [hunk 1]
```diff
@@ -2,40 +2,61 @@
...
       "type": "object",
       "properties": {
         "can_assign_matrix": {
           "type": "array",
           "items": {
             "type": "number"
           },
-          "description": "Assignment table for data association"
+          "description": "Assignment table for data association.",
+          "default": []
```
Comment (technolojin) [inline perception/autoware_object_merger/schema/data_association_matrix.schema.json:15]:
```suggestion
          "default": [
            0, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 0, 0, 1, 1, 1, 1,
            0, 0, 0, 0, 1, 1, 1, 1, 0, 0, 0, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0, 0, 0, 0, 1, 1, 1, 0, 0,
            0, 0, 0, 1, 1, 1
          ]
```

File: perception/autoware_object_merger/schema/data_association_matrix.schema.json [hunk 2]
```diff
@@ -2,40 +2,61 @@
...
         },
         "max_dist_matrix": {
           "type": "array",
           "items": {
             "type": "number"
           },
-          "description": "Maximum distance table for data association"
+          "description": "Maximum distance table for data association.",
+          "default": []
```
Comment (technolojin) [inline perception/autoware_object_merger/schema/data_association_matrix.schema.json:23]:
```suggestion
          "default": [
            4.0, 4.0, 5.0, 5.0, 5.0, 2.0, 2.0, 2.0, 4.0, 2.0, 5.0, 5.0, 5.0, 1.0, 1.0, 1.0, 5.0,
            5.0, 5.0, 5.0, 5.0, 1.0, 1.0, 1.0, 5.0, 5.0, 5.0, 5.0, 5.0, 1.0, 1.0, 1.0, 5.0, 5.0,
            5.0, 5.0, 5.0, 1.0, 1.0, 1.0, 2.0, 1.0, 1.0, 1.0, 1.0, 3.0, 3.0, 3.0, 2.0, 1.0, 1.0,
            1.0, 1.0, 3.0, 3.0, 3.0, 2.0, 1.0, 1.0, 1.0, 1.0, 3.0, 3.0, 2.0
          ]
```

File: perception/autoware_object_merger/schema/data_association_matrix.schema.json [hunk 3]
```diff
@@ -2,40 +2,61 @@
...
         "max_rad_matrix": {
           "type": "array",
           "items": {
-            "type": "number"
+            "type": "number",
+            "minimum": 0.0
           },
-          "description": "Maximum angle table for data association. If value is greater than pi, it will be ignored."
+          "description": "Maximum angle table for data association. If value is greater than pi, it will be ignored.",
+          "default": []
```
Comment (technolojin) [inline perception/autoware_object_merger/schema/data_association_matrix.schema.json:32]:
```suggestion
          "default": [
            3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 1.047, 1.047, 1.047, 1.047, 3.15,
            3.15, 3.15, 3.15, 1.047, 1.047, 1.047, 1.047, 3.15, 3.15, 3.15, 3.15, 1.047, 1.047,
            1.047, 1.047, 3.15, 3.15, 3.15, 3.15, 1.047, 1.047, 1.047, 1.047, 3.15, 3.15, 3.15,
            3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15,
            3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15, 3.15
          ]
```

File: perception/autoware_object_merger/schema/data_association_matrix.schema.json [hunk 4]
```diff
@@ -2,40 +2,61 @@
...
         "min_iou_matrix": {
           "type": "array",
           "items": {
-            "type": "number"
+            "type": "number",
+            "minimum": 0.0
           },
-          "description": "Minimum IoU threshold matrix for data association. If value is negative, it will be ignored."
+          "description": "Minimum IoU threshold matrix for data association. If value is negative, it will be ignored.",
+          "default": []
```
Comment (technolojin) [inline perception/autoware_object_merger/schema/data_association_matrix.schema.json:41]:
```suggestion
          "default": [
            0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.3, 0.2, 0.2, 0.2, 0.1, 0.1, 0.1, 0.1,
            0.2, 0.3, 0.3, 0.3, 0.1, 0.1, 0.1, 0.1, 0.2, 0.3, 0.3, 0.3, 0.1, 0.1, 0.1, 0.1, 0.2,
            0.3, 0.3, 0.3, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1,
            0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1, 0.1
          ],
```

**Review by technolojin — COMMENTED** [review]
I added some missing default values.

**Comment by technolojin** [comment]
I saw some tables are merged together because the markdown interpreted those as a table
```
{{ json_to_markdown("perception/autoware_object_merger/schema/object_association_merger.schema.json") }}
{{ json_to_markdown("perception/autoware_object_merger/schema/data_association_matrix.schema.json") }}
{{ json_to_markdown("perception/autoware_object_merger/schema/overlapped_judge.schema.json") }}
```

We need to set space between the script
```
{{ json_to_markdown("perception/autoware_object_merger/schema/object_association_merger.schema.json") }}

{{ json_to_markdown("perception/autoware_object_merger/schema/data_association_matrix.schema.json") }}

{{ json_to_markdown("perception/autoware_object_merger/schema/overlapped_judge.schema.json") }}
```

Can you implement this change in similar places?

- https://github.com/autowarefoundation/autoware.universe/blob/main/perception/autoware_probabilistic_occupancy_grid_map/README.md#parameters

- https://github.com/autowarefoundation/autoware.universe/tree/main/perception/autoware_object_merger#parameters

or maybe more

**Comment by vividf** [comment]
@technolojin 
Thanks for your suggestions!
They are fixed in https://github.com/autowarefoundation/autoware.universe/pull/9766/commits/e77c495c0fa52bc5ebef91c21dde45f37688d69b

**Review by technolojin — APPROVED** [review]
LGTM

**Review by YoshiRi — APPROVED** [review]
LGTM

**Pull request merged**